ui-tests/cypress/support/pageObjects/components/uxFragments/FragmentsPage.js

```diff
@@ -0,0 +1,105 @@
+import {htmlElements} from '../../WebElement.js';
+
+import AppContent from '../../app/AppContent.js';
+
+import AppPage         from '../../app/AppPage.js';
+import UXFragmentsPage from './UXFragmentsPage.js';
+
+export default class FragmentsPage extends AppContent {
+
+  codeInput             = `${htmlElements.input}[name="code"]`;
+  guiCodeInput          = `${htmlElements.textarea}[name="guiCode"]`;
+  guiCodeSelector       = `${htmlElements.a}[id="basic-tabs-tab-1"]`;
+  defaultGuiSelector    = `${htmlElements.a}[id="basic-tabs-tab-2"]`;
+  saveBtn               = `${htmlElements.button}[type=button]#saveopts`;
+  saveOption            = `${htmlElements.a}#regularSaveButton`;
+  saveAndContinueOption = `${htmlElements.a}#continueSaveButton`;
+  cancelBtn             = `${htmlElements.button}[class="pull-right btn btn-default"]`;
+
+  static saveButton(button, code = null) {
```

Comment:
why is it a static method?

Reply:
I thought a static method was used on a pageObject class, as opposed to other methods used on instantiated objects. 
After your explanation I understand the difference, thanks